src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/ISQLServerBulkRecordIssuesTest.java

```diff
@@ -445,4 +408,54 @@ public boolean next() throws SQLServerException {
         return false;
     }
 
+    @Override
+    public void addColumnMetadata(int positionInFile,
+            String name,
+            int jdbcType,
+            int precision,
+            int scale,
+            DateTimeFormatter dateTimeFormatter) throws SQLServerException {
+        // TODO Not Implemented
```

Comment:
Should we throw a "not implemented" exception when users try to use these APIs; like we do with sharding APIs.